src/HistoricalPlanList.tsx

```diff
@@ -74,24 +74,17 @@ export class HistoricalPlanList extends React.Component<HistoricalPlans, Histori
                             user_group: {planObject.user_group}
                           </Typography>
                           <Typography>
-                            exit_status: {planObject.exit_status}
+                            exit_status: {planObject.result.exit_status}
+                          </Typography>
+                          <Typography>
+                            run_uids: {JSON.stringify(planObject.result.run_uids)}
                           </Typography>
+                          <Previews runUid={planObject.result.run_uids[0]}/>
```

Comment:
What if there are more elements in `run_uids`?

Reply:
For now, I was just planning to support a single runUid, and add support in the future for multiple if we see that it is needed.